boards/arm/nrf9161dk_nrf9161/nrf9161dk_nrf9161_common.dtsi

```diff
@@ -230,16 +230,12 @@ arduino_spi: &spi3 {
 		slot0_ns_partition: partition@50000 {
 			label = "image-0-nonsecure";
 		};
-		slot1_partition: partition@80000 {
+		slot1_partition: partition@85000 {
 			label = "image-1";
 		};
-		slot1_ns_partition: partition@c0000 {
+		slot1_ns_partition: partition@c5000 {
 			label = "image-1-nonsecure";
 		};
-		scratch_partition: partition@f0000 {
-			label = "image-scratch";
-			reg = <0x000f0000 0xa000>;
-		};
 		storage_partition: partition@fa000 {
 			label = "storage";
 			reg = <0x000fa000 0x00006000>;
```

Comment:
Shouldn't this be done like in dc431a7397a629c9fb7e3a27a803d11bf1103389? @joerchan

Reply:
I suppose this depends on how you want to use the freed up space - I put it in the app image.

Reply:
As long as the TF-M partitions are placed correctly, they don't follow the configuration you put here, they are statically defined.

Reply:
Is this documented somewhere? It seems a bit hard to understand which partition config ends up being used if parts of the configuration are overwritten by build system magic.

Reply:
No. It's not overwritten, it's ignored ;) However in NCS the partition manager symbols are used. So this is only a zephyr restriction.